cirq-google/cirq_google/transformers/target_gatesets/sycamore_gateset_test.py

```diff
@@ -0,0 +1,351 @@
+0: ───[       │   │         ]───────────────────[       │         │    ]───────────────────×['ignore']───[       │             │    ]───────────────────────────[           [ 1: ───X───────@─── ]        ]───────────────────────────[       │         │    ]───────────────────X['ignore']───[           │    ]───────────────────
+      [ 1: ───×───ZZ^0.5─── ]['swap_rzz']       [ 1: ───ZZ^0.25───×─── ]['swap_rzz']       │             [ 2: ───ZZ^0.75───H───X─── ]['2q_component']           [           │                             ]                           [ 1: ───ZZ^0.15───×─── ]['swap_rzz']                     [ 1: ───Y───X─── ]['2q_component']
+      │                                         │                                          │             │                                                      [ 1: ───H───#2─────────────────────────── ]['2q_component']           │                                                        │
+      │                                         │                                          │             │                                                      │                                                                     │                                                        │
+1: ───#2────────────────────────────────────────#2─────────────────────────────────────────┼─────────────┼──────────────────────────────────────────────────────#2────────────────────────────────────────────────────────────@───────#2───────────────────────────────────────────────────────#2───────────────────────────────────
+                                                                                           │             │                                                                                                                    │
+2: ────────────────────────────────────────────────────────────────────────────────────────×─────────────#2───────────────────────────────────────────────────────────────────────────────────────────────────────────────────X─────────────────────────────────────────────────────────────────────────────────────────────────────
+''',
+    )
+
+
+# pylint: enable=line-too-long
+
+
+def test_merge_swap_rzz_and_2q_unitaries_raises_if_tags_sames():
+    with pytest.raises(ValueError, match="should be different"):
+        sycamore_gateset.merge_swap_rzz_and_2q_unitaries(
+            cirq.Circuit(),
+            merged_swap_rzz_tag='merged_component',
+            merged_2q_component_tag='merged_component',
+        )
+
+
+def test_sycamore_gateset_compiles_swap_zz():
+    qubits = cirq.LineQubit.range(3)
+
+    gamma = np.random.randn()
+    circuit1 = cirq.Circuit(
+        cirq.SWAP(qubits[0], qubits[1]),
+        cirq.Z(qubits[2]),
+        cirq.ZZ(qubits[0], qubits[1]) ** gamma,
```

Comment:
Does differing values of `gamma` change the the compiled circuit structure drastically? Wondering why you chose not to assert the compiled circuit against a hard-coded circuit.

Reply:
The tests were copied from previous implementation of convert_to_sycamore_gates.py and I don't think changing gamma to a hardcoded value would make much of a difference. 

The important thing to test is that swap + rzz does get compiled via the known compilations flow.